tests/unit/reconcile/query_builder/test_conf.py

```diff
@@ -58,6 +58,6 @@ def get_table_conf_all_options(self):
                 Transformation(column_name="s_phone", source="trim(s_phone)", target="trim(s_phone)"),
                 Transformation(column_name="s_name", source="trim(s_name)", target="trim(s_name)"),
             ],
-            thresholds=[Thresholds(column_name="s_acctbal", lower_bound="0", upper_bound="100", type="int")],
+            thresholds=[Thresholds(column_name="s_acctbal", lower_bound="0", upper_bound="100", type="integer")],
```

Comment:
how are you supporting other datatypes ?

Reply:
Now, we are supporting all the below data types available in databricks.
# NUMBER
    BIGINT
    INT
    SMALLINT
    TINYINT
    FLOAT
    DOUBLE
    DECIMAL
    # DATETMME
    DATE
    TIMESTAMP
    TIMESTAMP_NTZ

Reply:
No we support all numeric and temporal types available in sqlglot